packages/shopping-cart/src/pages/Products.js

```diff
@@ -9,8 +9,8 @@ export default function Products(props) {
   return(
     <div className="container">
       <div className="row">
-        {productsList.arrayOfProducts.map((product, index) =>
-          <Product product={product} key={index} authStatus={authStatus}/>
+        {productsList.arrayOfProducts.map((product) =>
```

Comment:
what will be shown when product list is empty while we are making api call to server ?